pages/common/xml-validate.md

```diff
@@ -0,0 +1,24 @@
+# xml validate
+
+> Validate XML documents.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Validate XML document(s) for well-formedness only:
```

Comment:
We could do the `on or more` trick in this page as well, but I'm not sure if the descriptions might get too long then.

Reply:
I'll look at it.